connexion/exceptions.py

```diff
@@ -95,3 +96,20 @@ def __init__(self, token_scopes, required_scopes, title=None, **kwargs):
         self.missing_scopes = required_scopes - token_scopes
 
         super(OAuthScopeProblem, self).__init__(title=title, **kwargs)
+
+
+class ExtraParameterProblem(ProblemException, BadRequest):
+    def __init__(self, formdata_parameters, query_parameters, title=None, description=None, **kwargs):
```

Comment:
We should separate this in two different Exceptions (one for formdata and another for query). The way it is now this exception means "there is a extra parameter in formdata or query or both, I am not sure". This is not good.